src/pages/filter/dtree/components/modals/components/modal-view-variants.tsx

```diff
@@ -21,6 +21,7 @@ import { TCondition } from '@service-providers/common/common.interface'
 import { fetchDsListAsync } from '@utils/TableModal/fetchDsListAsync'
 import { fetchJobStatusAsync } from '@utils/TableModal/fetchJobStatusAsync'
 import { ReturnedVariantsDataCy } from '../../../../../../components/data-testid/returned-variants'
+import modalsVisibilityStore from '../modals-visibility-store'
 
 const ModalView = styled.div`
```

Comment:
Maybe we can rid of styled components here

Reply:
this will be implemented in some @AlMaQntr PR